src/unstract/sdk/adapters/utils.py

```diff
@@ -45,15 +49,15 @@ def get_file_mime_type(input_file: Path) -> str:
         Returns:
             str: MIME type of the file
         """
-        input_file_mime = ""
-        with open(input_file, mode="rb") as input_file_obj:
-            sample_contents = input_file_obj.read(100)
-            input_file_mime = magic.from_buffer(sample_contents, mime=True)
-            input_file_obj.seek(0)
+        sample_contents = fs.read(path=input_file, mode="rb", length=100)
```

Comment:
Increasing the default `length` to a higher value, such as `1024`, would optimize the reading process by reducing the number of iterations.

Reply:
@muhammad-ali-e the idea is for this function to go away post file storage APIs are integrated and tested. Would prefer to not change this as this will be deleted in some time.